FRENCH/src/ch07-02-defining-modules-to-control-scope-and-privacy.md

```diff
@@ -0,0 +1,249 @@
+        fn ajouter_a_la_liste_attente() {}
+
+        fn installer_a_une_table() {}
+    }
+
+    mod service {
+        fn prendre_commande() {}
+
+        fn servir_commande() {}
+
+        fn encaisser() {}
+    }
+}
+```
+
+<!--
+<span class="caption">Listing 7-1: A `front_of_house` module containing other
+modules that then contain functions</span>
+-->
+
+<span class="caption">Encart 7-1 : Un module `salle_a_manger` qui contient
+d'autres modules qui contiennent eux-mêmes des fonctions</span>
+
+<!--
+We define a module by starting with the `mod` keyword and then specify the
+name of the module (in this case, `front_of_house`) and place curly brackets
+around the body of the module. Inside modules, we can have other modules, as in
+this case with the modules `hosting` and `serving`. Modules can also hold
+definitions for other items, such as structs, enums, constants, traits, or—as
+in Listing 7-1—functions.
+-->
+
+Nous définissons un module en commençant avec le mot-clé `mod` et nous précisons
+ensuite le nom du module (dans notre cas, `salle_a_manger`) et nous ajoutons des
+accolades autour du corps du module. Dans les modules, nous pouvons avoir
+d'autres modules, comme dans notre cas avec les modules `accueil` et `service`.
+Les modules peuvent aussi contenir des définitions pour d'autres éléments, comme
+des structures, des énumérations, des constantes, des traits, ou des fonctions
+(comme c'est le cas pour ces dernières dans l'encart 7-1).
+
+<!--
+By using modules, we can group related definitions together and name why
+they’re related. Programmers using this code would have an easier time finding
+the definitions they wanted to use because they could navigate the code based
+on the groups rather than having to read through all the definitions.
+Programmers adding new functionality to this code would know where to place the
+code to keep the program organized.
+-->
+
+Grâce aux modules, nous pouvons regrouper ensemble des définitions qui sont
+liées et donner un nom à ce lien. Les développeurs qui utiliseront ce code
+pourront plus facilement trouver les définitions dont ils ont besoin car ils
+peuvent parcourir le code en fonction des groupes plutôt que d'avoir à lire
+toutes les définitions. Les développeurs qui veulent rajouter des nouvelles
+fonctionnalités à ce code sauront maintenant où placer le code tout en gardant
+le programme organisé.
+
+<!--
+Earlier, we mentioned that *src/main.rs* and *src/lib.rs* are called crate
+roots. The reason for their name is that the contents of either of these two
+files form a module named `crate` at the root of the crate’s module structure,
+known as the *module tree*.
+-->
+
+Précédemment, nous avons dit que *src/main.rs* et *src/lib.rs* étaient des
+racines de crates. Nous les appelons ainsi car le contenu de chacun de ces
+deux fichiers constituent un module qui s'appelle `crate` à la racine de la
+structure du module de cette crate, aussi appelée *l'arborescence du module*.
```

Comment:
Becomes harder to follow. I'd suggest:
```MarkDown
deux fichiers constituent un module qui s'appelle `crate` à la racine de
*l'arborescence du module*, c'est-à-dire la structure des modules de cette crate

Reply:
You're probably right, `arborescence` speaks by itself. But on the other hand, this book is meant to be accessible to anyone... I'll leave the choice to you